### DISCUSSION TITLE
Is there a way to change the config so all the downloaded files goes into one folder instead of sub folders?

### DISCUSSION THREAD

**Discussion by pzrd** [UltimaHoarder/UltimaScraper#1105] [Q&A] [answered]
I've always just manually moved the Images and Videos folder from inside the Posts folder to the main model username folder and then dropped everything from Archived and Stories to the root folder, deleted those empty folders then sorted and moved them to Images and Videos accordingly. 

I've thoroughly read through the "Optional" config options, but I can't find what I'm looking for. Basically I'd like my folder to look like this:

modelname
 - Images
 - Videos
 - Metadata

Instead of how it is currently:

modelname
 - Archived
 -- Posts
 --- Free
 ---- Images
 ---- Videos
 - Posts 
 -- Free 
 --- Images
 --- Videos

Essentially removing folders like "archived" "posts" and "free" and just downloading every Image to Images and every video to Videos.

**Comment by UltimaHoarder** [comment, marked as answer]
`{username}\\{media_type}`

This is the format you're looking for

**Reply by pzrd** [reply]
Thank you! I realised it was right in front of me after posting this lmao. One last thing, is there a way to choose not to download the Avatars and Headers folders?

**Reply by UltimaHoarder** [reply]
> Thank you! I realised it was right in front of me after posting this lmao. One last thing, is there a way to choose not to download the Avatars and Headers folders?

Not at this time, but it will be an option.

**Reply by olivechicago** [reply]
> > Thank you! I realised it was right in front of me after posting this lmao. One last thing, is there a way to choose not to download the Avatars and Headers folders?
> 
> Not at this time, but it will be an option.

I'd love this option as well.  I was hoping that setting auto_media_choice to Video would do it... but no luck.
